R/outbreak_step.R

```diff
@@ -111,7 +111,10 @@ outbreak_step <- function(case_data,
     # time when new cases were exposed, a draw from generation time based on
     # infector's onset
     exposure = incubation_to_generation_time(
-      rep(onset, new_cases), event_probs$alpha
+      incubation_period_samples = rep(onset, new_cases),
+      exposure_time = rep(exposure, new_cases),
+      alpha = event_probs$alpha,
```

Comment:
noting that the order here is different to the function arguments (but as mentioned there I think it makes sense as is here)